test/utils/test-utils.js

```diff
@@ -247,6 +247,17 @@ const normalizeError = (output) => {
     return output.replace(/SyntaxError: .+/, 'SyntaxError: <error-message>').replace(/\s+at .+(}|\)|\d)/gs, '\n    at stack');
 };
 
+const normalizeCompileTime = (output) => {
+    return output.replace(/in \d+ ms/gm, 'in <compile time> ms').replace();
```

Comment:
explain your regexes :)

Reply:
yep better to add an explanation and some examples of what's being replaced